recipes-tag/pyzbar/run_test.py

```diff
@@ -0,0 +1,3 @@
+import pyzbar
+import pyzbar.pyzbar
+from pyzbar.pyzbar import decode
```

Comment:
I'm not familiar. Will this get automatically tested? Also why not just put it in the test import key of the meta.yml?

Reply:
Yes, it's automatically tested. The reason is that you cannot directly import it with `import pyzbar.pyzbar.decode`.

Reply:
I haven't found an example how you can include `from pyzbar.pyzbar import decode` into the `meta.yml` directly. If you encountered something like this, I'll be happy to get rid of the `run_test.py` file.

Reply:
Some docs about test files: https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html?highlight=run_test#test-section